scripts/weather.sh

```diff
@@ -18,14 +25,38 @@ display_weather()
 	else
 		display_weather='&m' # for metric system
 	fi
-	echo $(weather_information $display_weather)
+	weather_information=$(fetch_weather_information $display_weather)
+
+	weather_condition=$(echo $weather_information | awk '{print $1;}') # Sunny, Snow, etc
```

Comment:
I'm splitting the string in two to get the temperature and the weather condition, the latter is used to get the Unicode character. 
**wttr.in** returns something like this: `Sunny 36°C`, so the weather condition is `Sunny` and the temperature is `36°C`. 
For splitting the word I'm using `awk` but there's another approach transforming the word into an array and then accessing the words  using `array[0]` and `array[1]` but,  I don't know which of these is the best approach,

Reply:
why not use cut? i.e. `echo "Sunny 36°C" | cut -d ' ' -f 1` returns `Sunny`

Reply:
I'm quite new to bash so I don't know a lot of tools, but I'll change it to `cut`. I just found two bugs, wttr.in sometimes can return this: **"Partly cloudy +34°C"**. The first bug is that we split the string using a space as a delimiter, this won't work because when we try to get the temperature, instead of getting **+34°C** we will get **cloudy**. So, I was thinking of 
 doing this: 
```bash
# Split the word using the plus or minus sign
weather_condition=$(echo "Partly cloudy -34°C" | cut -d "+" -f 1 | cut -d "-" -f 1) 
temperature=$(echo "Partly cloudy -34°C" | cut -d "+" -f 2 | cut -d "-" -f 2)
```
Here  I'm using `cut` twice because I don't know a way of specifying two delimiters. So if you know a better approach just let me know.  
The second bug is when we try to get the Unicode character. If we use the same example from above (**"Partly cloudy +34°C"**), then this case won't work 
```bash
elif [[ (($weather_condition =~ 'Overcast') || ($weather_condition =~ 'Cloud')) ]]; then
```
because **cloudy** is in lowercase. Here I was thinking of transforming the variable `weather_condition` to lowercase, what do you think?

Reply:
Yes, I think that could be a good idea. Obviously the keywords are going to be a bit different because the verbiage used is different on wttr.in than the previous weather data site that was being scraped. Implement whatever works / makes the most sense to you.